app/dashboard/templates/base.html

```diff
@@ -79,6 +79,12 @@
                             <i class="fa fa-stethoscope"></i>&nbsp;Tests<small>&nbsp;<sup>&#946;</sup></small>
                         </a>
                     </li>
+                    <li id="li-co-test">
+                        <a id="co-tests-l" href=":3000"
```

Comment:
will we use same server with different port? If yes is the answer, you need to be a little bit more explicit here..`href` needs server ip/address and probably there are some approaches that could be used to solve the problem.

Reply:
No it won't apply to production, so we need to have `https://staging.kernelci.org:3000`.